awx/main/management/commands/inventory_import.py

```diff
@@ -500,7 +500,7 @@ def _delete_groups(self):
                 'Group "%s" from v1 API is not deleted by overwrite',
                 self.inventory_source.deprecated_group.name
             )
-            del_group_pks.discard(self.inventory_source.deprecated_group_id)
+        del_group_pks.discard(self.inventory_source.deprecated_group_id)
```

Comment:
this change in indent looks unintentional and could result in a discard of `None` which might blow up.

Reply:
You...are correct :)